source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -410,4 +417,9 @@ def findExtraOverlayClasses(obj, clsList):
 
 
 class WinTerminalUIA(EnhancedTermTypedCharSupport):
-	pass
+	def _get_caretMovementDetectionUsesEvents(self):
```

Comment:
Isn't this the default? Also, I don't think this needs to be a property.

Reply:
> Isn't this the default?

[Nope](https://github.com/nvaccess/nvda/blob/master/source/NVDAObjects/behaviors.py#L361).